homeassistant/components/twilio/__init__.py

```diff
@@ -47,7 +49,9 @@ async def handle_webhook(hass, webhook_id, request):
     data["webhook_id"] = webhook_id
     hass.bus.async_fire(RECEIVED_DATA, dict(data))
 
-    return TwiML().to_xml()
+    return Response(
+        status=HTTPStatus.OK, body=MessagingResponse().to_xml(), content_type="text/xml"
+    )
```

Comment:
Shouldn't this be handled in twilio instead?

Reply:
TwiML only generates a Response when the client is handling the send request. In this case we’re just trying to generate an appropriate body to return in the Webhook response so that twilio knows that we’ve acked and not to continue the messaging conversation. On second look though I think I could remove all of the twilio client code because it’s not used in this at all now.

Reply:
Never mind I can’t remove that code because it’s used by twilio_call and twilio_sms

Reply:
OK, so you're happy with this PR as is?

Reply:
Yes - this is my first commit to HA do I need to fix the import order since the it failed workflow?